Core/src/org/sleuthkit/autopsy/datamodel/accounts/Accounts.java

```diff
@@ -329,18 +331,35 @@ Long getCount(String accountType) {
             return types;
         }
 
+        /**
+         * Calculates the total count of accounts based on the values in the
+         * map.
+         *
+         * @return The total count.
+         */
+        Long getTotal() {
```

Comment:
Is this used?

Reply:
No.  I forgot to take it out.  It's gone now.